developerportal/templatetags/app_filters.py

```diff
@@ -24,6 +26,14 @@ def hex_to_rgb(hex_color, alpha=1):
     return f"rgb({r}, {g}, {b})" if a == 1 else f"rgb({r}, {g}, {b}, {a})"
 
 
+@register.filter(name="published")
+def published(items):
+    """Filters StreamField items to remove drafts Pages."""
```

Comment:
I appreciate this might be a pain to test in the context of a real StreamField, but any chance of a unit test for this (using simpler data structures for `item` perhaps)?